sqlserver/tests/test_metadata.py

```diff
@@ -23,6 +23,8 @@
 def dbm_instance(instance_docker):
     instance_docker['dbm'] = True
     instance_docker['min_collection_interval'] = 1
+    instance_docker['query_metrics'] = {'enabled': False}
+    instance_docker['procedure_metrics'] = {'enabled': False}
```

Comment:
should activity be disabled as well here? Maybe it makes sense to make a special `dbm_metrics` instance config that enables both procs and query metrics only / sets the collection intervals?